assets/js/renderer.js

```
@@ -1,186 +1,179 @@
+  let passphrase = document.getElementById('passphrase-input').value || undefined
+  let publicKey = await window.utils.generatePublicKey(privateKey, passphrase)
+  document.getElementById('public-key-text-area').value = publicKey
+
+  if (publicKey) {
+    change_buttons_disabled_state(false)
+  } else {
+    change_buttons_disabled_state(true)
+  }
 }
 
 const copyKey = async (input_key) => {
```

ESLint

[error] Identifier 'input_key' is not in camel case.

(camelcase)